packages/asset-swapper/test/dex_sampler_test.ts

```diff
@@ -1,657 +1,657 @@
+//             const uniswapV2ETHQuotes = [
+//                 expectedMakerFillAmounts.map(a => ({
+//                     source: ERC20BridgeSource.UniswapV2,
+//                     input: a,
+//                     output: a.times(ratesBySource[ERC20BridgeSource.UniswapV2]).integerValue(),
+//                     fillData: {
+//                         router: uniswapV2Router,
+//                         tokenAddressPath: [expectedTakerToken, wethAddress, expectedMakerToken],
+//                     },
+//                 })),
+//             ];
+//             //  extra quote for Uniswap V2, which provides a direct quote (tokenA -> tokenB) AND an ETH quote (tokenA -> ETH -> tokenB)
+//             expect(quotes).to.have.lengthOf(sources.length + 1);
+//             expect(quotes).to.deep.eq(expectedQuotes.concat(uniswapV2ETHQuotes));
+//         });
+//         describe('batched operations', () => {
+//             it('getLimitOrderFillableMakerAssetAmounts(), getLimitOrderFillableTakerAssetAmounts()', async () => {
+//                 const expectedFillableTakerAmounts = ORDERS.map(() => getRandomInteger(0, 100e18));
+//                 const expectedFillableMakerAmounts = ORDERS.map(() => getRandomInteger(0, 100e18));
+//                 const sampler = new MockSamplerContract({
+//                     getLimitOrderFillableMakerAssetAmounts: (orders, signatures) => {
+//                         expect(orders).to.deep.eq(SIMPLE_ORDERS);
+//                         expect(signatures).to.deep.eq(ORDERS.map(o => o.signature));
+//                         return expectedFillableMakerAmounts;
+//                     },
+//                     getLimitOrderFillableTakerAssetAmounts: (orders, signatures) => {
+//                         expect(orders).to.deep.eq(SIMPLE_ORDERS);
+//                         expect(signatures).to.deep.eq(ORDERS.map(o => o.signature));
+//                         return expectedFillableTakerAmounts;
+//                     },
+//                 });
+//                 const dexOrderSampler = new DexOrderSampler(
+//                     chainId,
+//                     sampler,
+//                     undefined,
+//                     undefined,
+//                     undefined,
+//                     undefined,
+//                     async () => undefined,
+//                 );
+//                 const [fillableMakerAmounts, fillableTakerAmounts] = await dexOrderSampler.executeAsync(
+//                     dexOrderSampler.getLimitOrderFillableMakerAmounts(ORDERS, exchangeProxyAddress),
+//                     dexOrderSampler.getLimitOrderFillableTakerAmounts(ORDERS, exchangeProxyAddress),
+//                 );
+//                 expect(fillableMakerAmounts).to.deep.eq(expectedFillableMakerAmounts);
+//                 expect(fillableTakerAmounts).to.deep.eq(expectedFillableTakerAmounts);
+//             });
+//         });
+//     });
+// });
+// // tslint:disable-next-line: max-file-line-count
```

Comment:
Who needs tests anyway 😂 🙈

Reply:
Oh I actually meant to delete this file lol. Yeah we need to do a testing sprint when the dust settles. So much of the code is changing so rapidly rn, and we were already extremely reliant on simbot, that I figure we're better off without them for the moment.